src/main/resources/application.yml

```diff
@@ -28,6 +28,8 @@ spring:
     hikari:
       # time in ms to wait for db connection
       initialization-fail-timeout: 1800000
+      data-source-properties:
+        reWriteBatchedInserts: true
```

Comment:
This enables batching on the DB itself (otherwise it breaks the batches back into separate inserts when actually doing the work)